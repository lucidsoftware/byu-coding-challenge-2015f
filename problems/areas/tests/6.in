4 2
10 6
2 8
0 6