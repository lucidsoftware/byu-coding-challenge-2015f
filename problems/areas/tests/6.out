14.6666666667
